### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by mediadeo** [chx/webform_product#7]
I can't get any of this to work, do you have some documentation? pls?

**Comment by mudasirafzal** [comment]
Yeah for me its not working, After adding the commerce product its not showing on form

**Comment by ChristopherMatthews** [comment]
Likewise, the README.text is quite confusing too. I submitted an issue here on Drupal.org https://www.drupal.org/project/webform_product/issues/3109399

**Comment by scubafly** [comment]
I'm missing how to use this as well. Tried to follow the steps from the readme but got stuck.
